serialization-jackson/src/main/resources/reference.conf

```diff
@@ -35,6 +35,17 @@ pekko.serialization.jackson {
   migrations {
   }
 
+  # Controls the Buffer Recycler Pool implementation used by Jackson.
+  # https://javadoc.io/static/com.fasterxml.jackson.core/jackson-core/2.16.2/com/fasterxml/jackson/core/util/JsonRecyclerPools.html
+  # The default is "thread-local" which is the same as the default in Jackson 2.16.
+  buffer-recycler {
+    # the supported values are "thread-local", "lock-free", "shared-lock-free", "concurrent-deque",
+    # "shared-concurrent-deque", "bounded"
+    pool-instance = "thread-local"
```

Comment:
it's a thread-local based? In https://github.com/eclipse-vertx/vert.x/pull/5084/ vert.x guy sent a VT friendly about this too, should we have that in mind?

Reply:
could you read https://github.com/FasterXML/jackson-module-scala/issues/672 ?

until Jackson has a sensible default, we will need to stick with the thread-local

I am a committer on the Jackson project. I know quite a bit about Jackson code.